pkg/api/v1/schema.go

```diff
@@ -25,28 +25,19 @@ package v1
  ATTENTION: Rerun code generators when comments on structs or fields are modified.
 */
 
-import (
-	"kubevirt.io/kubevirt/pkg/precond"
-)
-
 type DomainSpec struct {
-	Name    string   `json:"name"`
-	UUID    string   `json:"uuid,omitempty"`
```

Comment:
This is duplicating information already present in the 'VM' object metadata. Allowing the DomainSpec to be configured with name/uuid that differs from the 'VM' that owns it is a recipe for confusion & bugs. The designer just takes the name/uuid direct from the VM metadata when generating XML, since the code everywhere expects those to be used.